fastestimator/estimator/estimator.py

```diff
@@ -80,6 +82,11 @@ def _prepare_estimator(self):
             assert isinstance(trace, Trace)
             if isinstance(trace, ModelCheckpoint):
                 no_save_warning = False
+            if isinstance(trace, LRController):
+                trace.log_steps = self.log_steps
+                if trace.lr_schedule:
+                    trace.lr_schedule.total_epochs = self.epochs
```

Comment:
instead of manually setting this from the estimator, we should pass these variables in the on_begin trace dictionary. that way other users who have traces that want this info can get it too

Reply:
likewise with the log_steps info for all traces. better to minimize the amount of coupling between estimator and specific traces

Reply:
good idea, fixed